### DISCUSSION TITLE
How to determine what each icon means?

### DISCUSSION THREAD

**Discussion by mikedloss** [starship/starship#3233] [Q&A] [answered]
In my directory (name omitted), I have a long chain of `via`'s. How can I tell what each one means?
![image](https://user-images.githubusercontent.com/2460574/141506924-8fec9520-2c42-412d-99b4-bbbc6f6bf51b.png)

**Comment by davidkna** [comment, marked as answer]
You can run `starship explain` to get an explanation.

**Reply by mikedloss** [reply]
thank you for this!
